client/signup/steps/import-from/components/not-authorized/index.tsx

```diff
@@ -3,39 +3,38 @@ import { useI18n } from '@wordpress/react-i18n';
 import page from 'page';
 import React from 'react';
 import { getStepUrl } from 'calypso/signup/utils';
-import { GoToStep } from '../../../import/types';
 
 import './style.scss';
 
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 
 interface Props {
-	goToStep: GoToStep;
 	siteSlug: string;
 }
 
 const NotAuthorized: React.FunctionComponent< Props > = ( props ) => {
 	const { __ } = useI18n();
-	const { goToStep, siteSlug } = props;
+	const { siteSlug } = props;
 
 	/**
 	 ↓ Methods
 	 */
 	const backToStart = (): void => {
 		page( getStepUrl( 'importer', 'capture', '', '', { siteSlug } ) );
 	};
+	const backToIntent = (): void => {
+		page( getStepUrl( 'setup-site', 'intent', '', '', { siteSlug } ) );
+	};
 
 	return (
 		<div className="import-layout__center">
 			<div className="import__header">
 				<div className="import__heading  import__heading-center">
-					<Title>{ __( 'Your are not authorized to import content' ) }</Title>
+					<Title>{ __( 'You are not authorized to import content' ) }</Title>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.